### DISCUSSION TITLE
3070 ti (LHR) on ETH

### DISCUSSION THREAD

**Discussion by qwaszxman** [NebuTech/NBMiner#514] [General]
I have a 3070ti (LHR) which always gets hit by the LHR lock whilst mining ETH on version 39.5

The maximum hashrate i have been able to get is ~44MH/s but using other the similar settings on other miners its able to get ~54MH/s at the same power draw.

I am using HiveOS and i've tried 

removing the PowerLimiter
Upping the CoreClock
Using both LockCoreClock and +/- CoreClock
Lowering the MemoryClock
I've also tried various LHR values and have had to settle on 60 as this is currently the most stable

These are the settings im using currently which give me 44MH/s

LHR 60
LockCoreClock1050 
MemoryClock1400 
PL 160

On other miners im able to increase the MemoryClock to 2000 and get a hashrate of ~54MH

Does anyone have any ideas?

**Comment by FlokiMiner** [comment]
I have 3 x 3070ti's  ETH 54-55MHs on Hive. LHR 68  NB miner 39.5

I generally cringe when setting PL high, i really aim for efficiency, but it seems with 70ti's you need to allow some headroom up to Power limit 180w.   Very diff to what i have found with 3060's I usually trim the PL down until the LHR lock doesn't kick in on those.  But 70ti doesn't seem to work that way.

I am using 
Core 1050
Mem 2850 - 3200 (seems to need tweaking per card for best rate)
PL 180 (if i set lower it kills the hashrate)

Power will jump around 120-180 , seems to not affect the hashrate much. This is mentioned in the LHR low power mode notes.

Also make sure you let it settle for a few minutes every time you tweak on OC.  Best of Luck~!
